rm-services-rh-stub/app.py

```diff
@@ -90,10 +91,22 @@ def postcaseevent(caseid):
 
 @app.route('/iacs/<iaccode>')
 def get_case_details_from_iac(iaccode):
+    """ Route handler to deal with getting details from the iac service.
+    The iaccode passed in must be either vvvvvvvvvvvv or wwwwwwwwwwww
+    otherwise a 404 status will be returned to indicate an invalid UAC"""
     my_iac_stub = Iac_Stub(iaccode)
-    data = Iac_Stub.get_iac_stub_for_all(my_iac_stub)
-
-    return Response(data, mimetype='application/json')
+    data_dict = Iac_Stub.get_iac_stub_for_all(my_iac_stub)
+    #test to see if first key in dict is error or not
+    data_keys = list(data_dict)
+    firstKey = str(data_keys[0])
+    logging.info("The first key in the list is: " + firstKey)
```

Comment:
I don't know if renaming this "firstKey" to "jsonResponseStatusCodeKey" would make it more readable, but firstKey should be more specific as an identifier to what we need

Reply:
Thanks for your comment Alin. I named it firstKey following the KISS principle; it is the first key in a list of keys, named data_keys, as indicated by the code. The reason why the key is taken is to test whether or not it is named "error", which becomes clear in the following lines of code.